app/src/main/java/zapsolutions/zap/transactionHistory/listItems/LnInvoiceViewHolder.java

```diff
@@ -0,0 +1,109 @@
+package zapsolutions.zap.transactionHistory.listItems;
+
+import android.view.View;
+import android.widget.TextView;
+import androidx.core.content.ContextCompat;
+import zapsolutions.zap.R;
+import zapsolutions.zap.util.MonetaryUtil;
+import zapsolutions.zap.util.Wallet;
+
+
+public class LnInvoiceViewHolder extends TransactionViewHolder {
+
+    private TextView mAmount;
+
+    public LnInvoiceViewHolder(View v) {
+        super(v);
+        mAmount = v.findViewById(R.id.transactionAmount);
+    }
+
+    public void bindLnInvoiceItem(LnInvoiceItem lnInvoiceItem) {
+
+        // Standard state. This prevents list entries to get mixed states because of recycling of the ViewHolder.
+        setSuccessState(true);
+
+        // Set fee
+        setFee(0, false);
+
+        // Set time of Day
+        setTimeOfDay(lnInvoiceItem.mCreationDate);
+
+        // Set description
+        if (lnInvoiceItem.getInvoice().getMemo().equals("")) {
+            setSecondaryDescription("", false);
+        } else {
+            setSecondaryDescription(lnInvoiceItem.getInvoice().getMemo(), true);
+        }
+
+
```

Comment:
Can we remove some of the extra lines in the files where its not really necessary.